src/test/groovy/com/contrastsecurity/ContrastPluginTest.groovy

```diff
@@ -65,4 +67,24 @@ class ContrastPluginTest {
         assertEquals(argLine, actualArgLine)
     }
 
+    @Test
+    public void testComputeAppVersionQualifier() {
+
+        String travisBuildNumber = System.getenv("TRAVIS_BUILD_NUMBER")
```

Comment:
Let's rework this test.

Unit tests shouldn't depend on environment variables being set. Set the environment in your test

Create a test where TRAVIS_BUILD_NUMBER (or the corresponding variable in the plugin) is set. Call `computeAppVersionQualifier` and test that the output of `computeAppVersionQualifier` is what we would expect.

Do the same for:
* CIRCLE_BUILD_NUM
* None set at all
* Both TRAVIS_BUILD_NUMBER and CIRCLE_BUILD_NUM set

Reply:
@gmoore Done, added more tests